src/ast/mod.rs

```diff
@@ -4556,13 +4556,16 @@ pub enum FunctionArgOperator {
     Equals,
     /// function(arg1 => value1)
     RightArrow,
+    /// function(arg1 := value1)
+    DuckAssignment,
```

Comment:
I suggest a general name like Assignment

Reply:
I renamed both Token and FunctionArg to Assignment , since [snowflake uses it too](https://github.com/gstvg/sqlparser-rs/blob/e8ad6af36b7019cfee181101fb47463a944d64eb/src/parser/mod.rs#L4143)

What do you think @universalmind303 ?

If is best to split the Token rename to another PR I can do it, just let me know, thanks!